pkg/version/version.go

```diff
@@ -38,8 +38,8 @@ var (
 	// It is customary to bump the version by incrementing the numeral after
 	// the `dev` tag. The SQL migration script name must correspond to the /new/ version.
 
-	Promscale                           = "0.7.0"
-	PrevReleaseVersion                  = "0.7.0-beta.1"
+	Promscale                           = "0.7.1-dev.0"
+	PrevReleaseVersion                  = "0.7.0"
```

Comment:
It would be nice if we would use a `VERSION` file and load this data dynamically.

And for development version we could just add short git sha suffix to note promscale is built from a different commit.

With such changes PRs like this would be unnecessary.

It is fairly easy to derive from git history if the commit

Reply:
Can you file an issue for this?